Diary/Scene/DiaryList/Controller/DiaryListViewController.swift

```diff
@@ -20,11 +21,18 @@ final class DiaryListViewController: UIViewController {
     private var diaryCollectionView: UICollectionView?
     private var dataSource: UICollectionViewDiffableDataSource<Section, Diary>?
     private var diaryCoreManager: DiaryCoreDataManager?
+    private let locationManager = CLLocationManager()
+    private var icon: String?
     
     // MARK: - life cycles
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        locationManager.delegate = self
+        locationManager.requestWhenInUseAuthorization()
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.startUpdatingLocation()
```

Comment:
```suggestion
        setupLocationManager()
```
=> viewDidLoad안 다른 메소드들(`setupView`, `setupDataSource`, `addObserver`)처럼 메소드 추상화 단계가 맞도록 
위의 예시처럼 함수분리 해주면 좋을 듯 합니다~

Reply:
현재 GPS 현재 위치를 못찾거나 혹은 위치 허용을 하지 않았을때 일기장을 아예 못쓰던데, 어떤 경우에도 일기장은 쓰이도록 구현해주면 더 좋을 듯 합니다. 현재 날씨를 모르더라도 일기장은 쓸 수 있게끔 해주는게 더 좋을 듯 해요 👍